.evergreen/config/build-variants.yml

```diff
@@ -0,0 +1,87 @@
+#
+# Build variants to build the driver - these are run for all operating systems and PHP versions we support
```

Comment:
Were all of these variants copied from PHPC despite the fact that we don't actually test on them for PHPLIB? Does it make sense to do so or would you rather strip this down to the minimum required?

Reply:
Yes - these effectively allows us to test that running `pecl install` works as expected. Tests are only run on a smaller set of variants.